core/tests/protractor_utils/general.js

```diff
@@ -256,6 +256,52 @@ var navigateToTopicsAndSkillsDashboardPage = async function() {
   await waitFor.pageToFullyLoad();
 };
 
+var goOffline = async function() {
```

Comment:
The comment explaining the upload and download speeds should go with the first occurrence. I see two options:

1. Put goOnline above goOffline
2. In goOffline explain what download_throughput and upload_throughput mean. In goOnline, explain where the numbers come from (since in goOffline the 0s are pretty self-explanatory)

Reply:
Done.